library/core/src/ops/arith.rs

```diff
@@ -426,6 +444,12 @@ pub trait Div<Rhs = Self> {
     type Output;
 
     /// Performs the `/` operation.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// assert_eq!(12 / 2, 6);
```

Comment:
How about `12 / 0`?

Reply:
?? 12/0 is a compile time error: https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=750633198d7a3b2f68deffac99eed104

Reply:
Exactly, I think it would be good to have another example to show that it will fail to compile.